lib/linguist/languages.yml

```diff
@@ -2053,6 +2080,37 @@ Idris:
   ace_mode: text
   tm_scope: source.idris
   language_id: 165
+Ignore List:
+  type: data
+  group: INI
+  aliases:
+  - ignore
+  - gitignore
+  - git-ignore
+  extensions:
+  - ".gitignore"
+  filenames:
+  - ".atomignore"
+  - ".babelignore"
+  - ".bzrignore"
+  - ".coffeelintignore"
+  - ".cvsignore"
+  - ".dockerignore"
+  - ".eslintignore"
+  - ".gitignore"
+  - ".nodemonignore"
+  - ".npmignore"
+  - ".prettierignore"
+  - ".stylelintignore"
+  - ".vscodeignore"
+  - gitignore
+  - gitignore-global
+  - gitignore_global
```

Comment:
You did not include a link with in-the-wild usage for that filename, but [I think it's okay](https://github.com/search?utf8=%E2%9C%93&q=filename%3Agitignore_global+NOT+nothack&type=Code).

Reply:
Search results were the same with a hyphen or an underscore. 😠 Check it.

Reply:
Ah, right. It looks like there are [only 132 `gitignore-global` files](https://github.com/search?p=14&q=filename%3Agitignore-global+NOT+nothack&type=Code) though (`gitignore-global` come first in search results and then it's just `gitignore_global` files). It's probably a single file by repository, so usage might be good enough...?

Reply:
Yep, most definitely. I can't imagine anybody needing more than one global `gitignore` file. 😜